cli/commands/nodes.go

```diff
@@ -12,17 +12,17 @@ import (
 func nodeStatus(node api.Node) string {
 	if node.Connected {
 		return "online"
-	} else {
-		return "offline"
```

Comment:
Yes I agree with putting the special case first. Regarding the idea of breaking out / it being an inconsistent style to other conditional statements within this codebase; the other issues may have caused other side-effects because of the scope, unlike this one. This will satisfy the linter, so would be more, "the Go way" - however, being the maintainer, if you'd feel better the way it was, I can simply revert this, and keep the special case first. Just let me know 😄